// Bug happens on JVM , JVM -Xuse-ir
//File: tmp/tmp0.kt

fun box()  =
        try {} catch (K: Exception) {
try {} catch (e: Exception) {
fun f5()  {
class Foo
}
}
fun f5()   =
try {
class Foo
} catch (e: Exception) {}
}